pylint/checkers/similar.py

```diff
@@ -405,7 +736,10 @@ class SimilarChecker(BaseChecker, Similar, MapReduceMixin):
     def __init__(self, linter=None):
         BaseChecker.__init__(self, linter)
         Similar.__init__(
-            self, min_lines=4, ignore_comments=True, ignore_docstrings=True
+            self,
+            min_lines=DEFAULT_MIN_SIMILARITY_LINE,
```

Comment:
Shouldn't this be the real value from the configuration here ? Maybe it's the reason for https://github.com/PyCQA/pylint/pull/4565/files#r650381290 ?

Reply:
Yes i think you are right. I also added the values taken from configuration for `ignore_imports` and `ignore_signatures`.